src/sum.ts

```diff
@@ -0,0 +1,3 @@
+export function sum(x: number, y: number) {
```

Comment:
Looks like some boilerplate that can be removed.

Reply:
Cleaned up